### PR TITLE
[#12282] Instructor sessions page: Fix create feedback session card header

### LINKED ISSUE #12282: Instructor sessions page: Fix create feedback session card header
The create feedback session card header breaks at small screen widths.

At <800px:
Screenshot 2023-03-28 at 15 32 02

At mobile breakpoints (<767px):
Screenshot 2023-03-28 at 15 32 30

There are multiple ways to fix this, including but not limited to:
- Splitting the components into two rows at small screen widths
- Reducing the space between the dropdown and the info button

### PR DESCRIPTION
**Pull request by vaheaslanyan**
Fixes #12282 

**PR Checklist**

Ensure that you have:
- [x] Read and understood our PR guideline: https://teammates.github.io/teammates/process.html#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [x] Added/updated tests, if changes in functionality were involved
- [x] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

**Comment by vaheaslanyan** [comment]
I got it looking like this:

> 992px
992up

< 992px
upto992

< 576px
upto576

Let me know if anything needs to be changed. Thank you!

**Comment by vaheaslanyan** [comment]
Failing test seems to be comparing html snapshots which was obviously changed, can we update the snapshots? Or is there anything else I should do?

**Comment by samuelfangjw** [comment]
Yes please update the snapshots

**Comment by vaheaslanyan** [comment]
> Yes please update the snapshots

it does not give me an option to do that in watch mode, could it be an access issue or am I not looking in the right place?

**Comment by samuelfangjw** [comment]
Try running the tests first, option should be there when the test fails due to snapshots not matching.

**Comment by vaheaslanyan** [comment]
> Try running the tests first, option should be there when the test fails due to snapshots not matching.

You are correct, thank you for your help!

File: src/web/app/components/session-edit-form/session-edit-form.component.scss [hunk 1]
```diff
@@ -42,3 +42,28 @@
...
   color: gray;
 }
+
+.create-new-dropdown-div {
+  display: flex;
+}
+
+.session-heading-text {
+  display: flex;
+  text-align: left;
```
Comment (EuniceSim142) [inline src/web/app/components/session-edit-form/session-edit-form.component.scss:52]: i don't think this is necessary as you've already set the left margin and padding to 0.

File: src/web/app/components/session-edit-form/session-edit-form.component.html [hunk 2]
```diff
@@ -4,28 +4,26 @@
...
           </div>
-          <div class="col-9 col-md-8">
+          <div class="col-12 col-sm-9 create-new-dropdown-div">
             <select id="session-type" class="form-control form-select" [ngModel]="model.templateSessionName" (ngModelChange)="triggerModelChange('templateSessionName', $event)">
               <option *ngFor="let templateSession of templateSessions" [ngValue]="templateSession.name">{{ templateSession.name }}</option>
             </select>
-          </div>
-          <div class="col-1 col-md-1">
-            <a type="button" class="btn btn-link" tmRouterLink="/web/instructor/help" [queryParams]="{ questionId: 'sessions', section: 'sessions' }" target="_blank" aria-label="Help for feedback sessions"><i class="fas fa-info-circle"></i></a>
+            <a type="button" class="btn btn-link session-selector-info-btn" tmRouterLink="/web/instructor/help" [queryParams]="{ questionId: 'sessions', section: 'sessions' }" target="_blank" aria-label="Help for feedback sessions"><i class="fas fa-info-circle"></i></a>
```
Comment (EuniceSim142) [inline src/web/app/components/session-edit-form/session-edit-form.component.html:16]: There is no class selector named `session-selector-info-btn`.

File: src/web/app/pages-instructor/instructor-sessions-page/__snapshots__/instructor-sessions-page.component.spec.ts.snap [hunk 3]
```diff
@@ -463,13 +461,9 @@ exports[`InstructorSessionsPageComponent should snap when new session form is ex
...
                       </option>
                     </select>
-                  </div>
-                  <div
-                    class="col-1 col-md-1"
-                  >
                     <a
                       aria-label="Help for feedback sessions"
-                      class="btn btn-link"
+                      class="btn btn-link session-selector-info-btn"
```
Comment (EuniceSim142) [inline src/web/app/pages-instructor/instructor-sessions-page/__snapshots__/instructor-sessions-page.component.spec.ts.snap:466]: Same here. No class selector named `session-selector-info-btn`.

**Review by EuniceSim142 — COMMENTED** [review]
Just some minor nits, layout looks good!

**Comment by vaheaslanyan** [comment]
Thank you for the review @EuniceSim142, I made all the adjustments. Still got to look into why the e2e tests are failing, then I'll push it up.

**Comment by vaheaslanyan** [comment]
Ready to merge

**Review by EuniceSim142 — APPROVED** [review]
lgtm, thanks for contributing!

**Comment by jasonqiu212** [comment]
@vaheaslanyan Thanks for contributing!

@samuelfangjw @zhaojj2209 Would it be possible to re-run the E2E tests? The failing test passes on my local machine.

**Review by jasonqiu212 — COMMENTED** [review]
LGTM!

**Comment by jasonqiu212** [comment]
@samuelfangjw @zhaojj2209 Would it be possible to re-run the failing E2E and component tests? The tests pass on my local machine. Thank you!

**Pull request merged**